docs/data/rigid.tfm

```diff
@@ -0,0 +1,5 @@
+#Insight Transform File V1.0
+#Transform 0
+Transform: Rigid2DTransform_double_2_2
+Parameters: 0 0 0
```

Comment:
Perhaps a non-identity transform should be used to demonstrate the effect?

Reply:
We do not use the contents of the transformation in the documentation.